src/main/java/Duke.java

```diff
@@ -1,10 +1,125 @@
+
+// import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+
+    public static void listTask(Task[] list) {
+        if (list[0] == null) {
+            System.out.println("You have no tasks in your list yet!");
+            return;
+        }
+
+        int count = 1;
+        for (Task item : list) {
+            if (item != null) {
+                System.out.println(count + ". " + item.describeTask());
+                count++;
+            } else {
+                continue;
+            }
+        }
+
+        System.out.println("\nYou have " + (count - 1) + " tasks in the list.\n");
+    }
+
+    public static String getFirstWord(String s) {
+        String[] words = s.split(" ");
+        return words[0];
+    }
+
+    public static String getSecondWord(String s) {
+        int index = s.indexOf(" ");
+        String sub = s.substring(index + 1);
+        return sub;
+    }
+
+    public static void markTask(Task[] list, int index) {
+        list[index - 1].markAsDone();
+        System.out.println("Nice! I've marked this task as done:\n" + list[index - 1].describeTask() + "\n");
+    }
+
+    public static void unmarkTask(Task[] list, int index) {
+        list[index - 1].unmarkAsDone();
+        System.out.println("OK, Ive marked this task as not done yet:\n" + list[index - 1].describeTask() + "\n");
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+        Scanner in = new Scanner(System.in);
+        boolean isComplete = false;
+
+        while (!isComplete) {
+
+            String line = in.nextLine();
+            String firstWord = getFirstWord(line);
+            Task[] taskList = Task.getTaskList();
+
+            switch (firstWord) {
+            case "mark":
+                markTask(taskList, Integer.parseInt(getSecondWord(line)));
+                break;
+            case "unmark":
+                unmarkTask(taskList, Integer.parseInt(getSecondWord(line)));
+                break;
+            case "list":
+                listTask(taskList);
+                break;
+            case "todo":
+                getTodoDetails(line);
+                break;
+            case "deadline":
+                getDealineDetails(line);
```

Comment:
Should the method getDealineDetails be spelt as getDeadlineDetails instead? (line 72 and line 99)
```suggestion
                getDeadlineDetails(line);
```